### ISSUE TITLE
platformio-ide-terminal no longer installs on new Atom version 1.52.0

### ISSUE THREAD

**Issue by dougwyu** [platformio/platformio-atom-ide-terminal#924]
- [ ] I have reset Atom to defaults prior to submitting report.
- [x ] I have not reset Atom to defaults prior to submitting report.
### Description

platformio-ide-terminal stops working on new Atom (1.52.0).  

### Steps to reproduce

1. i updated Atom
2. platformio-ide-terminal threw an installation error (error messages below)

**Expected behavior:** 
platformio-ide-terminal should have kept on working

**Actual behavior:** 
error message upon install:

Installing “platformio-ide-terminal@2.10.0” failed.Hide output…

> node-pty-prebuilt-multiarch@0.9.0-beta21.legacy install /private/var/folders/4g/4wpmdzbd7tj0cwctcd99wz680000gp/T/apm-install-dir-2020919-88197-l99pqc.upj2e/node_modules/platformio-ide-terminal/node_modules/node-pty-prebuilt-multiarch
> prebuild-install || node scripts/install.js

prebuild-install WARN install No prebuilt binaries found (target=6.1.12 runtime=electron arch=x64 libc= platform=darwin)
No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.
No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.
No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.

gyp: No Xcode or CLT version detected!
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:200:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/Applications/Atom.app/Contents/Resources/app/apm/bin/node" "/Applications/Atom.app/Contents/Resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /private/var/folders/4g/4wpmdzbd7tj0cwctcd99wz680000gp/T/apm-install-dir-2020919-88197-l99pqc.upj2e/node_modules/platformio-ide-terminal/node_modules/node-pty-prebuilt-multiarch
gyp ERR! node -v v12.4.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open '/private/var/folders/4g/4wpmdzbd7tj0cwctcd99wz680000gp/T/apm-install-dir-2020919-88197-l99pqc.upj2e/package.json'
npm WARN apm-install-dir-2020919-88197-l99pqc.upj2e No description
npm WARN apm-install-dir-2020919-88197-l99pqc.upj2e No repository field.
npm WARN apm-install-dir-2020919-88197-l99pqc.upj2e No README data
npm WARN apm-install-dir-2020919-88197-l99pqc.upj2e No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-pty-prebuilt-multiarch@0.9.0-beta21.legacy install: `prebuild-install || node scripts/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-pty-prebuilt-multiarch@0.9.0-beta21.legacy install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Negorashi2011/.atom/.apm/_logs/2020-10-19T07_10_06_974Z-debug.log`

**Reproduces how often:** 
always

### Versions

```
> atom --version
Atom    : 1.52.0
Electron: 6.1.12
Chrome  : 76.0.3809.146
Node    : 12.4.0
```
```
> apm --version
apm  2.5.2
npm  6.14.8
node 12.4.0 x64
atom 1.52.0
python 3.8.3
git 2.28.0
```
OS name and version: macOS 10.15.7
Platformio-ide-terminal version: 2.10.0

### Additional Information

Although the error message says that macOS CLT cannot be found, i have CLT installed.  

I installed XCode. Same error message.

**Comment by dojo-technology** [comment]
same problem here

**Comment by snedigar** [comment]
same here

**Comment by Leo24790** [comment]
same

**Comment by ayanezcloud** [comment]
Same here. My favorite terminal stopped working a few days ago and cannot be reinstalled. Any news or clues to solve it?

**Comment by mitsuhisaT** [comment]
me too
please see issue #543

**Comment by JeSuisUnCaillou** [comment]
[Terminus](https://atom.io/packages/terminus) seems to be working, if anyone is looking for a quick alternative. It's a fork of platformio-ide-terminal.

**Comment by dougwyu** [comment]
Terminus works (on macOS)!

If you want to use `ctrl-enter` instead of `ctrl-i` to insert selected text (i.e. to send a command from a text file into the terminal and run it), add this command to the end of your keymap.cson file:

`'atom-workspace atom-text-editor:not([mini])': 'ctrl-enter': 'terminus:insert-selected-text'`

the keymap.cson file is opened by going to Atom/Keymap...

**Comment by winnies26** [comment]
Same here

**Comment by HuangStanley050** [comment]
same issue

**Comment by mjberg** [comment]
same issue

**Comment by kike1209** [comment]
Same issue here

**Comment by tarfoot** [comment]
I was able to get this to work by rebuilding the package. I made sure that I had Xcode installed. (I had to **launch Xcode** in order to complete the installation. Then I:

1. uninstalled Platform-IDE-Terminal
2. Quit and restarted Atom
3. Re-installed Platform-IDE-Terminal
4. Quit and restarted Atom
5. Clicked on the little red ladybug in the lower-right status bar.
6. Clicked on the "Rebuild Package" button to rebuild Platform-IDE-Terminal
7. Quit and restarted Atom

Back in action!
